ui/guest/src/react/MoveModeZoneMenu.tsx

```diff
@@ -56,16 +104,20 @@ export function MoveModeZoneMenu(props: MoveModeZoneMenuProps) {
           <HighlightOffRoundedIcon />
         </UltraStyledIconButton>
       </Tooltip>
-      <Tooltip title="Move up/left (← or ↑)">
-        <UltraStyledIconButton size="small" onClick={onMoveUp}>
-          <ArrowUpwardRoundedIcon />
-        </UltraStyledIconButton>
-      </Tooltip>
-      <Tooltip title="Move down/right (→ or ↓)">
-        <UltraStyledIconButton size="small" onClick={onMoveDown}>
-          <ArrowDownwardRoundedIcon />
-        </UltraStyledIconButton>
-      </Tooltip>
+      {index !== 0 && (
+        <Tooltip title="Move up/left (← or ↑)">
+          <UltraStyledIconButton size="small" onClick={onMoveUp}>
+            <ArrowUpwardRoundedIcon />
+          </UltraStyledIconButton>
+        </Tooltip>
+      )}
+      {index < dropzoneChildrenLength - 1 && (
```

Comment:
For clarity, please extract these conditions to vars above (`isFirstItem`, `isLastItem`). We should also have `isOnlyItem` and hide both buttons if the item is there by itself.